packages/image/types/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.png' {}
```

Comment:
should we remove this file now that `?generate` is gone?

Reply:
I believe Vite already contains those, so yeah